share/harvesters/io_osf.py

```diff
@@ -16,6 +16,10 @@ class OSFHarvester(BaseHarvester):
     VERSION = 1
 
     def build_url(self, start_date, end_date, path, query_params):
```

Comment:
Looks like this is based on an old develop, before some things were moved to `future-release/2.15.0`. Could you resolve conflicts?